packages/sysutils/busybox/scripts/getedid

```diff
@@ -0,0 +1,254 @@
+#!/bin/sh
+#
+#
+# exit 1 = unsupported GPU
+# exit 2 = dual boot system
+# exit 3 = no backup file, therefore no restore
+# exit 4 = extlinux.conf and syslinux.cfg are available
+# exit 5 = changes are already made either to extlinux.conf or syslinux.cfg
+# exit 6 = xorg.conf already exists in /storage/.config
+# exit 7 = no xorg.conf in /storage/.config
+
+
+# Help message and usage explanation
+help() {
+echo "This script generates a custom EDID depending on your GPU"
+echo ""
+echo "To check which GPU you are using, use: ./getedid gpu"
+echo ""
+echo "To create a custon EDID, just use this script like: ./getedid start"
+echo ""
+echo "If you already used that script and want to temporarly get thing back like it was before use: ./getedid restore"
+echo ""
+echo "If you don't want to use the created EDID file anymore use: ./getedid delete"
+echo ""
+echo "If you hardware has changed and you want to have a custom EDID again, then you have to delete the old EDID first."
+echo "For this use: ./getedid delete "
+echo "And then use: ./getedid start"
+}
+
+
+# check for GPU and store string
+check_gpu() {
+  if lspci | grep -i vga | grep -i -q intel ; then
+    gpu="intel"
+  elif lspci | grep -i vga | grep -i -q nvidia; then
+    gpu="nvidia"
+  else
+    echo "GPU is not supported"
+    exit 1
+  fi
+}
+
+
+# run this first if the user already has a custom EDID but want to create a new one (TV or AVR change)
+del_edid() {
+  if [ "$gpu" = "intel" ]; then
+    check_file
+    if [ -f "$file".old ];  then
+      mount_rw
+      rm "$file"
+      mv "$file".old "$file"
+      mount_ro
+      rm -fr /storage/cpio
+      rm -fr /storage/.config/firmware
+      sys_reboot
+    else
+      echo "You don't have a backup file for $file. You didn't used this script to create the custom EDID"
+      echo "Therefore we can't be sure the script working prooerly. Exiting"
+      exit 3
+    fi
+    rm -f /storage/.config/xorg.conf
+    rm -f /storage/.config/xorg.le.backup
+    sys_reboot
+  fi   
+}
+
+
+# run main script depending on the GPU
+run() {
+  if [ "$gpu" = "intel" ]; then
+    intel 
+  elif [ "$gpu" = "nvidia" ]; then
+    nvidia
+  fi
+}
+
+
+# mounting /flash to rw
+mount_rw() {
+  mount -o remout,rw /flash
+}
+
+
+# mounting /flash to ro
+mount_ro() {
+  mount -o remount,ro /flash
+}
+
+
+# remount /flash to ro and reboot
+sys_reboot() {
+  mount_ro
+  echo "The system is rebooting in 15 seconds"
+  sleep 15
+  reboot
+}
+
+
+# check syslinux.cfg and/or extlinux.conf
+check_file() {
+  # check boot system
+  boot="$([ -d /sys/firmware/efi ] && echo UEFI || echo BIOS)"
+  if [ "$boot" = "BIOS" ]; then
+    sys_path="/flash"
+  elif [ "$boot" = "UEFI" ]; then
+    sys_path="/flash/EFI/BOOT"
+  fi
+
+  #check which file is available
+  if [ -f "$sys_path"/syslinux.cfg ] && [ -f "$sys_path"/extlinux.conf ];then
```

Comment:
`"$sys_path"/syslinux.cfg` ->`"$sys_path/syslinux.cfg"`
`"$sys_path"/extlinux.conf` ->`"$sys_path/extlinux.conf"`

and the same wherever this has been repeated. Presumably it works as it is, but I find it a little odd.